.ddev/config.yaml

```
@@ -0,0 +1,39 @@
+# DDev configuration
+# @see https://ddev.readthedocs.io/en/latest/users/extend/config_yaml/
+
+# Update to match your project name. Using "drupal-skeleton" would make the site
+# accessible at 'drupal-skeleton.ddev.site'.
+name: drupal-skeleton
+
+# Use 'docroot' for Acquia, or 'web' for Pantheon or Platform.sh.
+docroot: web
+
+# Add any multisite subdomains to this array. Adding "example" will make this environment
+# available at "example.ddev.site".
+additional_hostnames: []
+
+# Add extra fully qualified domains here. This is useful for transitioning from a vagrant
+# environment to a ddev environment.
+additional_fqdns: []
+
+# The PHP version should match the project's hosting environment.
+php_version: "7.3"
+
+# It may make sense to turn this off once the project is set up.
+disable_settings_management: false
```

Should we let ddev manage settings until the configuration of the project is ready?
```suggestion
disable_settings_management: true
```

---

This is a double negative, so the current setting of `false` allows ddev to manage the `settings.php` :)